### DISCUSSION TITLE
Which tools would you recommend for Vocaluxe?

### DISCUSSION THREAD

**Discussion by marwin89** [Vocaluxe/Vocaluxe#615] [Q&A] [answered]
I only know [Ultrastar Manager](https://github.com/UltraStar-Deluxe/UltraStar-Manager) and [Ultrastar Creator](https://github.com/UltraStar-Deluxe/UltraStar-Creator).

But it seems that there are more helpful tools for song management and song creating in the making.

**Comment by Stefan1200-de** [comment, marked as answer]
I also like to use: [Yass – Karaoke Editor](https://yass-along.com)

**Comment by marwin89** [comment]
I just discovered [USDB Syncer](https://github.com/bohning/usdb_syncer) recently. First Release in may 2023.
And also [UltraSinger](https://github.com/rakuri255/UltraSinger) (under developement).
